scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -83,11 +72,12 @@ class ScanCode(
     name: String,
     config: ScannerConfiguration
 ) : LocalScanner(name, config) {
-    class Factory : AbstractScannerFactory<ScanCode>("ScanCode") {
+    class Factory : AbstractScannerFactory<ScanCode>(SCANNER_NAME) {
```

Comment:
Commit message: Please stick to the imperative, "[...] has been added, too." ->"[...] is added, too."

Reply:
All commit messages have been reworked.